app/views/professional/proposals/show.html.erb

```diff
@@ -3,7 +3,7 @@
     <div class="col-md-8">
       <h3>Proposal ID: <%= @proposal.id %></h3>
       <h4>Project Name: <%= @proposal.project.name %></h4>
-      <h4>Project Requester: <%= @proposal.project.user.name %></h4>
+      <h4>Project Requester: <%=link_to @proposal.project.user.name, requester_path(@proposal.project.user) %></h4>
```

Comment:
When we go back to refactor, this is probably a good opportunity to make a method! Something that demonstrates that this refers to the requester's name.